src/Symfony/Component/Console/Exception/CommandNotFoundException.php

```diff
@@ -26,7 +26,7 @@ class CommandNotFoundException extends \InvalidArgumentException implements Exce
      * @param int        $code         Exception code
      * @param \Throwable $previous     Previous exception used for the exception chaining
      */
-    public function __construct(string $message, array $alternatives = [], int $code = 0, \Throwable $previous = null)
+    public function __construct(string $message, array $alternatives = [], int $code = 11, \Throwable $previous = null)
```

Comment:
i would only exclude this type of error from the general error type.

not sure anticapting the exceptions below is needed in practice, but this one i can imagine to allow for failure based on `$? -eq 11`.